docs/vectorizer_guide.md

```diff
@@ -167,12 +171,12 @@ SELECT ai.create_vectorizer(
 );
 ```
 
-The default format string is simply '$chunk'.
+The default format string is simply `$chunk`.
 
-### Indexing
+## Improve query performance on your Vectorizer
 
-The vectorizer can create a vector index on the embedding column to improve
-query performance. By default, a vectorscale index is created after 100,000
+To improve query performance, you create a vector index on the embedding column 
+in a Vectorizer. By default, a vectorscale index is created after 100,000
```

Comment:
I'm not convinced about this change, IMO it doesn't clarify anything.

There are three concepts at play here:
1. Creating an index on vector data is good (and sometimes necessary).
2. The Vectorizer will automatically create an index when 100k rows of vector data are present.
3. This behaviour is configurable.

This edit feels like it mixes the concepts.

Reply:
Updated the description to match what your comment.